plotille/_graphs.py

```diff
@@ -25,13 +25,59 @@
 
 from math import log
 import os
+from typing import List, Optional
 
 from ._colors import color
 from ._figure import Figure
 from ._input_formatter import InputFormatter
 from ._util import hist as compute_hist
 
 
+def hbar(counts, labels=Optional[List[str]], width=80, log_scale=False, linesep=os.linesep,
```

Comment:
What is the difference between this and hist_aggregated? Is it only the bins=len(counts) part?

Reply:
Exactly. Just needed a horizontal bar chart for something.